### PR TITLE
Fetch capabilities from singleServer if set

### LINKED ISSUE #531: Capabilities fetched from other server in single server mode
screenshot from 2018-05-04 11-57-05

screenshot from 2018-05-04 11-58-12

> Note: This also happens when session/cookies are cleaned...

@magopian can you look at it please?

It blocks deployment of kinto-dist 7.0.1 in PROD (https://bugzilla.mozilla.org/show_bug.cgi?id=1457150)

### PR DESCRIPTION
**Pull request by magopian**
Fixes #531

### CODE REVIEW

File: test/index_test.js [hunk 1]
```diff
@@ -28,10 +28,10 @@ describe("KintoAdmin", () => {
       buckets: [{}],
       serverInfo: {},
     };
-    const createKintoAdmin = () =>
+    const createKintoAdmin = (settings = {}) =>
       createComponent(KintoAdmin, {
         plugins: [],
-        settings: {},
+        settings: settings,
```
Comment (leplatrem) [inline test/index_test.js]: nit: with ES6 `{settings}` is enough

File: src/index.js [hunk 2]
```diff
@@ -44,7 +45,10 @@ export default class KintoAdmin extends Component<Props> {
       this.store.dispatch(
         sessionActions.getServerInfo({
           authType: "anonymous",
-          server: DEFAULT_KINTO_SERVER,
+          server:
+            settings.singleServer ||
+            (history && history[0]) ||
```
Comment (leplatrem) [inline src/index.js]: Should we have a test for that too?

Comment (leplatrem) [reply src/index.js]: I'm just a bit puzzle that it looks very similar to https://github.com/Kinto/kinto-admin/blob/master/src/components/AuthForm.js#L315
I know that we don't want to refactor it now, but can we at least add a comment to «connect» the two places for future contributions?

File: test/index_test.js (hunk 1 above)
Comment (magopian) [reply test/index_test.js]: you mean just `settings` right? Like so:

```javascript
    const createKintoAdmin = (settings = {}) =>
      createComponent(KintoAdmin, {
        plugins: [],
        settings,
      });
```

Comment (leplatrem) [reply test/index_test.js]: yes

File: src/index.js (hunk 2 above)
Comment (magopian) [reply src/index.js]: I've just refactored it in `utils/getServerByPriority`

**Comment by magopian** [comment]
@leplatrem I believe I've addressed all your comments, please let me know if there's anything else you'd like changed!

**Pull request merged**